sel_phase_linac.py

```diff
@@ -34,13 +34,8 @@ def __init__(
         )
         os.makedirs(os.path.dirname(self.logfile), exist_ok=True)
 
-        self.file_handler = logging.FileHandler(self.logfile, mode="w")
-
-        formatter = logging.Formatter(
-            fmt="%(asctime)s %(levelname)-8s %(message)s",
-            datefmt="%Y-%m-%d %H:%M:%S",
-        )
-        self.file_handler.setFormatter(formatter)
+        self.file_handler = logging.FileHandler(self.logfile, mode="a")
```

Comment:
If always appending to the same file, do we have to worry about filesize at some point? I guess it's just text...?

Reply:
I didn't think overwriting the file every time we restarted the process was a great idea but I honestly don't know how much info is too much

Reply:
I agree that overwriting every time is also not great. Maybe we start out like this and observe for a while: How fast does the logfile grow, and how unwieldy is trying to look at a logfile that has thousands and thousands of lines. Then we can start thinking about clever ways to break them up (if necessary)